db/seeds/utils.js

```diff
@@ -20,3 +23,12 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
```

Comment:
Interesting util function! I wouldn't have put this in the seeds/utils file though, it would be a good thing to make a new one (as this isn't a util function used for your seed)